.gitignore

```diff
@@ -2,3 +2,4 @@
 /composer.lock
 /.php_cs
 /.php_cs.cache
+.idea/
```

Comment:
You shouldn't add IDE-specific files to a project `.gitignore` file: you should add it to a global `.gitignore` file (see for example [here](https://stackoverflow.com/questions/7335420/global-git-ignore)).

Reply:
Right. However the JetBrains IDEs seems to be a kind of gold standard nowadays so we tend to prefer excluding such files in the project specifict `.gitignore`.

Thank you for the link anyway.

Adriano C.

Reply:
Well, not everybody uses phpstorm )

In public open source projects, in order to have a `.gitignore` file that suits most used IDEs, you should have these lines:

```
.buildpath
.idea
.project
.settings
.vscode
DS_Store
thumbs.db
__MACOSX
```

That for every project. And you should update all your public projects in case a new IDE comes up.

It's much easier to let users manage their own global .gitignore file ;)